azure-pipelines.yml

```diff
@@ -0,0 +1,103 @@
+# Build a Universal Windows Platform project using Visual Studio.
+# Add steps that test and distribute an app, save build artifacts, and more:
+# https://aka.ms/yaml
+
+trigger:
+  - master
+
+pool:
+  vmImage: "windows-latest"
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Build
+        timeoutInMinutes: 30
+        strategy:
+          maxParallel: 8
+          matrix:
+            Debug x86:
+              BuildConfiguration: Debug
+              BuildPlatform: x86
+              BuildPlatformDir: Win32
+              PublishMui: false
+              RunVSTest: true
+            Debug x64:
+              BuildConfiguration: Debug
+              BuildPlatform: x64
+              BuildPlatformDir: x64
+              PublishMui: false
+              RunVSTest: true
+            Debug ARM:
+              BuildConfiguration: Debug
+              BuildPlatform: ARM
+              BuildPlatformDir: Arm
+              PublishMui: false
+              RunVSTest: false
+            Debug ARM64:
+              BuildConfiguration: Debug
+              BuildPlatform: ARM64
+              BuildPlatformDir: Arm64
+              PublishMui: false
+              RunVSTest: false
+            Release x86:
+              BuildConfiguration: Release
+              BuildPlatform: x86
+              BuildPlatformDir: Win32
+              PublishMui: true
+              RunVSTest: true
+            Release x64:
+              BuildConfiguration: Release
+              BuildPlatform: x64
+              BuildPlatformDir: x64
+              PublishMui: false
+              RunVSTest: true
+            Release ARM:
+              BuildConfiguration: Release
+              BuildPlatform: ARM
+              BuildPlatformDir: Arm
+              PublishMui: false
+              RunVSTest: false
+            Release ARM64:
+              BuildConfiguration: Release
+              BuildPlatform: ARM64
+              BuildPlatformDir: Arm64
+              PublishMui: false
+              RunVSTest: false
+        steps:
+          - checkout: self
+            clean: true
+            lfs: true
+
+          - task: NuGetInstaller@0
+            displayName: "NuGet Restore"
+            inputs:
+              solution: BasicXrApp.sln
+              verbosity: Detailed
+
+          - task: NuGetInstaller@0
+            displayName: "NuGet Restore"
+            inputs:
+              solution: Samples.sln
+              verbosity: Detailed
+
+          - task: VSBuild@1
+            displayName: "Build Samples.sln"
+            inputs:
+              solution: Samples.sln
+              vsVersion: "16.0"
+              msbuildArgs: '/fl /flp:LogFile=$(System.DefaultWorkingDirectory)\bin\Logs\MsBuild_$(BuildConfiguration)_$(BuildPlatform).log;Verbosity=diagnostic'
+              platform: $(BuildPlatform)
+              configuration: $(BuildConfiguration)
+              maximumCpuCount: true
+
+          - task: VSBuild@1
+            displayName: "Build BasicXrApp.sln"
```

Comment:
Will this be merged into Samples.sln?

Reply:
So far, this BasicXrApp is stand alone, not using XrSceneLib.  Maybe keep it as is so it's the simple MVP sample without extra dependencies.

Reply:
Ah ok sounds good